src/main/java/org/apache/commons/imaging/ImageParser.java

```diff
@@ -13,6 +13,8 @@
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
+ *
+ * Changed 2015 by Michael Gross, mgmechanics@mgmechanics.de
```

Comment:
This has to be removed from all files. Please file an [ICLA](http://www.apache.org/licenses/).